src/app/shopping-list/shopping-list.component.html

```diff
@@ -14,14 +14,18 @@ <h1 class="margin-left" i18n="@@itemsInStore">Items in store</h1>
     <!-- Expanded Content Column - The detail row is made up of this one column that spans across all columns -->
     <ng-container matColumnDef="expandedDetail">
       <td mat-cell *matCellDef="let element; let i = index" [attr.colspan]="displayedColumns.length">
-          <div class="example-element-detail"
-              [@detailExpand]="element == expandedElement ? 'expanded' : 'collapsed'">
-              <mat-form-field floatLabel="never" appearance="fill">
-                  <mat-label i18n="@@typeAmountOfItems">Type amount of items</mat-label>
-                  <input matInput type="number" [(ngModel)]="amountOfItemsToAdd" min="0" max="{{element.quantity}}">
-              </mat-form-field>
-              <button color="primary" mat-raised-button (click)="addItemsToCart(element, ConvertStringToNumber(amountOfItemsToAdd))" i18n="@@addToCart">Add to cart</button>
-          </div>
+        <div class="example-element-detail" [@detailExpand]="element == expandedElement ? 'expanded' : 'collapsed'">
+        <table>
+          <tr>
+            <mat-form-field floatLabel="never" appearance="fill">
+                <mat-label i18n="@@typeAmountOfItems">Type amount of items</mat-label>
+                <input matInput type="number" [(ngModel)]="amountOfItemsToAdd" min="0" max="{{element.quantity}}">
+            </mat-form-field>
+            <button color="primary" mat-raised-button (click)="addItemsToCart(element, ConvertStringToNumber(amountOfItemsToAdd))" i18n="@@addToCart">Add to cart</button>
+          </tr>
+          <tr>{{element.description}}</tr>
+        </table>
```

Comment:
Po pierwsze brakuje tagu <td>, a po drugie to po co dawać całą tabelkę? Nie prościej zrobić to na divach i ustawić im width: 100% ? Nawet dodałem taką globalną klasę .maxWidth.